### ISSUE TITLE
Классификация: оценка качества

### ISSUE THREAD

**Issue by Kapatsa** [romichxd/STAT-ML#18]
@Loulaan 

https://github.com/romichxd/STAT-ML/blob/6efd5e236182be12d6e5c422b9ec0328157dbfa5/Supervised%20learning%20pt2/Report/Report.tex#L109

А разве не  \textit{recall} = ${TP}\over{TP+FN}$?

https://github.com/romichxd/STAT-ML/blob/6efd5e236182be12d6e5c422b9ec0328157dbfa5/Supervised%20learning%20pt2/Report/Report.tex#L110

И здесь \textit{precision} = ${TP}\over{TP+FP}$?

**Comment by neg99** [comment]
Да, Дейвид прав. Сверьтесь, например, с  https://en.wikipedia.org/wiki/Precision_and_recall
 
=========
 
  
>Среда, 12 января 2022, 19:18 +03:00 от David Kapatsa ***@***.***>:
> 
> 
***@***.***

**Comment by Loulaan** [comment]
Поправил
